Automattic/wp-calypso
# Stats: Add a plan usage section on the Stats Traffic page

Related to #84485

## Proposed Changes

* Add the Stats plan usage section gated behind the feature flag `stats/plan-usage`.
* Use props `usage`, `limit`, `daysToReset` and `overLimitMonths` to determine the usage status and upgrade text.

## Testing Instructions



* Spin this change with the local Calypso.
* Navigate to **Stats** > **Traffic** page with the feature flag `?flags=stats/plan-usage`.
* Ensure the plan usage section aligns with the design by modifying those parameters.

### Past and current usage all under the limit
<img width="804" alt="截圖 2023-11-28 下午11 21 37" src="https://github.com/Automattic/wp-calypso/assets/6869813/c50bf4d3-f6a5-451b-856c-9910068d90e7">

### Usage of the last month exceeds the limit
<img width="804" alt="截圖 2023-11-28 下午11 21 30" src="https://github.com/Automattic/wp-calypso/assets/6869813/0c3ec563-0371-4498-8ee6-bf5d6ba15cbf">

### The past two or more consecutive months of usage exceed the limit
<img width="815" alt="截圖 2023-11-28 下午11 45 32" src="https://github.com/Automattic/wp-calypso/assets/6869813/269728f4-7d82-4929-a7b3-8b0e9c4d739c">

### Current month usage over the limit under any conditions
<img width="816" alt="截圖 2023-11-28 下午11 21 14" src="https://github.com/Automattic/wp-calypso/assets/6869813/4aee57f6-0204-45f3-9603-a539dbd95e10">

## Pre-merge Checklist
